EIPS/eip-3475.md

```diff
@@ -2,16 +2,15 @@
 eip: 3475
 title: Multiple Callable Bonds Standard
 description: A standard interface for contract that manage multiple bonds. This standard will allow multiple LP token contract to store more information
-author: Yu LIU (@yuliu-debond), Waroon D. (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
+author: Yu LIU (@yuliu-debond), Waroon D. (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold Edoumou (@Edoumou), Toufic Batrice (@toufic0710)
 discussions-to: https://ethereum-magicians.org/t/eip-3475-multiple-callable-bonds-standard/8691
-status: Review
+status: Last Call
+last-call-deadline: 2022-06-30
```

Comment:
Unfortunately this'll need to be bumped out a bit further, since I predict a couple more rounds of editing... Maybe:

```suggestion
last-call-deadline: 2022-07-08
```

Reply:
true , in fact we will be having an PeePAnEIP session on 19th July , so idk whether should we finish the deliberation before this session or afterwards.